src/ui.js

```diff
@@ -26,9 +26,10 @@ export const makeTable = async () => {
       company[COMPANY_NAME_FIELD_NAME],
       company[STATUS_FIELD_NAME],
       company[CREATED_AT_FIELD_NAME],
-      company[REVENUE_YTD_FIELD_NAME],
+      new Intl.NumberFormat('fr-CA').format(company[REVENUE_YTD_FIELD_NAME]),
       company[ACCOUNT_EXECUTIVE_FIELD_NAME]
     );
+    console.log(company.revenue_ytd)
```

Comment:
I understand that you are using logging for debugging purposes. However when you are done, you should remove all logging statements before committing the code, because your commits go to production - and production code should not have any unnecessary logging or any other leftovers from some engineer who was doing debugging.